build-libs/tutorial-redirects.js

```diff
@@ -0,0 +1,24 @@
+async function getTutorialRedirects() {
+	const url = new URL(
+		'/pages/redirects',
+		'http://127.0.0.1:5000'
```

Comment:
TODO - remove this and uncomment the base url env. Was using this for local testing with the whole setup.